src/averageAges.js

```diff
@@ -14,48 +14,83 @@
  *
  * @return {number}
  */
+const getSum = (a, b) => a + b;
+
+const getSex = (people, sex) => people.filter(person => person.sex === sex);
```

Comment:
```suggestion
const getBySex = ...;
```
here, you don't get sex, you get people according to their sex, so consider giving proper names